### DISCUSSION TITLE
Welcome to ramp.xde Discussions!

### DISCUSSION THREAD

**Discussion by smitdave** [dd-harp/ramp.xds#68] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smitdave** [comment]
Hello, Juliet. Go ahead and paste questions here.

**Reply by Jnsumba** [reply]
![DTS_models_equilbrium](https://github.com/user-attachments/assets/589b4930-9b2e-4bf6-aa75-94a6142d3c22)
![DTS_models_equilbrium_pdf](https://github.com/user-attachments/assets/e05ca5ad-6d8f-4964-9c86-81b588d6b12e)
steady states for dts models
